recipes/abseil/all/conanfile.py

```diff
@@ -147,5 +147,6 @@ def package_info(self):
         ]
         if self.settings.os == "Linux":
             self.cpp_info.system_libs.append("pthread")
+        self.cpp_info.names["cmake"] = "absl"
```

Comment:
```suggestion
```
The `cmake` generator is not used to create the `FincAbsl.cmake` or `AbslConfig.cmake` scripts.

Reply:
Done.